External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -234,4 +277,28 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 end
 
 
+%% ====================================================================================
+%% 8. Create JSON sidecar
+if bSaveJson
+    % a. First, we load the reference sidecar JSON file, if it exists
+    if bCopyOrigJson && xASL_exist(pathOrigJson, 'file')
+        json = xASL_io_ReadJson(pathOrigJson);
+    elseif bCopyOrigJson
+        warning([pathOrigJson ' did not exist']);
+    else
+        json = struct([]);

```

Comment:
This won't work. 
try 
json = struct([])
json.test = 1
you have to change this back to struct or struct()

Reply:
Fixed in 67d6fdaccd87237de371db497c72181e2c8b1c8a